app/src/components/InfoInput/InfoInput.module.css

```diff
@@ -0,0 +1,66 @@
+h2 {
+    text-align: left;
+    position: relative;
+    top: 50px;
+    left: 1%;
+    margin: 0px;
+    font-family: Oswald, sans-serif;
+    font-size: 48px;
+    width: 99%;
+}
+
+.container {
+    position: relative;
+    left: 5%;
+    margin-top: 50px;
+    font-family: Oswald, sans-serif;
+    font-size: 36px;
+    width: 95%;
+}
+
+.pageLabel {
+    display: inline-block;
+}
+
+.pageItem {
+    position: relative;
+    display: block;
+    padding-left: 35px;
+    margin-bottom: 15px;
+    cursor: pointer;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    width: -moz-fit-content;
+    width: fit-content;
+}
+
+.pageItem input {
+    position: absolute;
+    opacity: 0;
+    cursor: pointer;
+    height: 0;
+    width: 0;
+}

```

Comment:
So you're not actually using this in your InfoInput component, so it's not actually styling any of the input fields with the text boxes. I think you probably should, though, so probably adding something like

```css
.container input {
    font-family: Oswald, sans-serif;
```

With maybe some other styling in there. I would also make one for the `textarea` to match the fonts. My only other suggestions are really just styling. I think you could throw a `margin-left: 10px;` or something similar in the `.container input {..}` "object" above. One last thing should be adding `vertical-align: middle;` to these css classes for `input` and `textarea` to align the text with the text boxes.